web/gds-user-ui/src/components/Collaborators/index.tsx

```diff
@@ -130,13 +129,22 @@ const CollaboratorsSection: React.FC = () => {
     onClose();
   };
   return (
-    <FormLayout overflowX={'auto'}>
+    <Stack 
```

Comment:
`FormLayout` has set styles and I didn't want to change them in that component since it's used all throughout the UI. Using some of the style here on a `Stack` element allows for more flexibility.